doc: update known issues for NCS v2.5.0 (#12657)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

  Currently, this issue occurs for Sidewalk v1.14 libraries, and it will be fixed in a future version.

.. rst-class:: v2-5-0

KRKNWK-17035: Sensor monitor uplink messages are lost when the notification period is longer than 30 seconds
  If the notification period is set to longer than 30 seconds, sensor monitor uplink messages are lost.

  **Workaround:** The notification period is set to 15 seconds by default.

.. rst-class:: v2-5-0

KRKNWK-14583: Bus fault after flash, before the :file:`Nordic_MFG.hex` data flash
  For sub-GHz samples, when the :file:`Nordic_MFG.hex` file is missing, the device throws a hard fault during initializing the Sidewalk stack.
  Proper error handling will be implemented, but the temporary solution is to write a manufacturing hex to the device and reset it.

.. rst-class:: v2-5-0

KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
  The non-resolvable private address (NRPA) cannot be set in the connectable mode for Bluetooth LE.
  Currently, there is no workaround for this issue.

Bluetooth® LE
=============

The issues in this section are related to :ref:`ug_ble_controller`.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer: These 4 issues need to be moved under  Bluetooth mesh.

Author: This is now fixed.

NCSDK-23087: Unsolicited Time Status rewrites periodic publishing TTL to zero forever
  The time model specification mandates publishing unsolicited Time Status messages with TTL ``0``.
  The implementation rewrites the TTL field to **zero** but does not write the initial value back, losing the initial value.

Workaround: Configure initial TTL value after an unsolicited Time Status is sent.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-23220: The Heartbeat Publication Status message may be malformed after provisioning
  After provisioning and obtaining composition data, reading the Heartbeat Publication and the Heartbeat Publication Status will contain garbage in **NetKeyIndex** field.
  The reason for this is that the field was not initially cleared.

.. rst-class:: v2-5-0

NCSDK-23308: Setting storage causes the device to reboot in the event of a clean operation
  For non-secure builds, whenever a flash erase while setting storage happens, it causes the device to reboot.

>   The same issue can be seen where stored model data is large enough and changes that cause defragging or cleaning.

Reviewer 2: ```suggestion
  The same issue can be seen where stored model data is large enough and changes cause defragging or cleaning.
```

Author: Updated all the comments. Please check.

  The error reproduces rarely (once per a few days).
  The device resets automatically in the release mode, however, in the debug mode it needs to be reset manually.
  Currently, this issue occurs for Sidewalk v1.14 libraries, and it will be fixed in a future version.

.. rst-class:: v2-5-0

KRKNWK-17035: Sensor monitor uplink messages are lost when the notification period is longer than 30 seconds
  If the notification period is set to longer than 30 seconds, sensor monitor uplink messages are lost.

  **Workaround:** The notification period is set to 15 seconds by default.

.. rst-class:: v2-5-0

KRKNWK-14583: Bus fault after flash, before the :file:`Nordic_MFG.hex` data flash
  For sub-GHz samples, when the :file:`Nordic_MFG.hex` file is missing, the device throws a hard fault during initializing the Sidewalk stack.
  Proper error handling will be implemented, but the temporary solution is to write a manufacturing hex to the device and reset it.

.. rst-class:: v2-5-0

KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
  The non-resolvable private address (NRPA) cannot be set in the connectable mode for Bluetooth LE.
  Currently, there is no workaround for this issue.

> Bluetooth® LE

Reviewer: There is 4 new known issus:

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-21625: Advertisements of Mesh GATT services are not stopped by :c:func:`bt_mesh_suspend` and not resumed by :c:func:`bt_mesh_resume
:c:func:`bt_mesh_suspend` and :c:func:`bt_mesh_resume does not work in conjunction with bt_disable() and bt_enable().

**Workaround:** To disable node identity advertisement use bt_mesh_subnet_node_id_set instead.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-23087: Unsolicited Time Status rewrites periodic publishing TTL to zero forever
Time model specification mandates to publish unsolicited Time Status messages with ttl 0. The implementation rewrites the TTL field to zero, but does not write the initial value back then losing the initial value.

**Workaround:** Configure initial TTL value after an unsolicited Time Status is sent.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-23220: The Heartbeat Publication Status message may be malformed after provisioning
After provisioning and obtaining Composition Data, reading the Heartbeat Publication and the Heartbeat Publication Status will contains garbage in NetKeyIndex field. The reason for this is that the field is not initially cleared.

.. rst-class:: v2-5-0

NCSDK-23308: Setting Storage causing device to reboot in case of clean operation
For non-secure builds, whenever a flash erase on setting storage happens, it causes device to reboot. The same issue can be seen where stored model data is big enough and changes that causes defragging/cleaning. EMDS partition is not effected by this.

Author: Updated. Please check.

Reviewer 2: All of these should probably go under Bluetooth mesh section. Right @Balaklaka?

Reviewer: Yes, they should be under Bluetooth mesh

>   **Workaround:** Do not enable the :kconfig:option:`CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE` option.

Author: Updated. Please check @annwoj and @ktaborowski

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1

> KRKNWK-17718: Failure at TC-DGSW-1.1 Matter Certification test.

Reviewer 3: ```suggestion
KRKNWK-17718: Failure at TC-DGSW-1.1 Matter Certification test
```

  The issue happens due to a wrong Software Diagnostic cluster feature map with features enabled by default.

  **Workaround:** Set the default value of the Software Diagnostic cluster feature map to 0 using the ZAP Tool and regenerate files.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

KRKNWK-17699: Failure at TC-BINFO-1.1 Matter Certification test for the lock sample.
  The issue happens due to a noncompliant Basic Information cluster revision for Matter 1.1.0 in the lock sample.

  **Workaround:** Set the default value of the Basic Information cluster revision to 1 in :file:`samples/matter/lock/src/lock.zap` using the ZAP Tool and regenerate files.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

> KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup.

Reviewer 3: ```suggestion
KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup
```

  This can happen because of the Thread interface being unnecessarily reset and can result in the TC-CNET-4.10 Matter Certification test failing.

  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``3b2d8e1367d9055a78d72365323cfbf60e054975``).

.. rst-class:: v2-5-0 v2-4-2 v2.4.1 v2-4-0 v2-3-0 v2-2-0

> KRKNWK-17535: The application core can crash on nRF5340 after the OTA firmware update finishes if the factory data module is enabled.

Reviewer 3: ```suggestion
KRKNWK-17535: The application core can crash on nRF5340 after the OTA firmware update finishes if the factory data module is enabled
```

Matter
======

> There are following new known issues to be added for the Matter:

Reviewer 4: I think this line should not be added, as it is just part of my github comment :)

Author: It was a copy past issue :). Updated.

> The issues in this section are related to the :ref:`ug_matter` protocol.

Reviewer 4: There are following new known issues to be added for the Matter:
```suggestion
The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-5-0

KRKNWK-17864: When using Wi-Fi Low Power Mode, the communication with the device may not work after it re-connects to the newly respawned Wi-Fi network
  The communication with Matter over Wi-Fi device sometimes does not work after it re-connects to the Wi-Fi network.
  The issue is visible in case of re-connection to the newly respawned Wi-Fi network, triggered by rebooting the Wi-Fi access point.
  The problem was not observed for re-connection triggered by loosing the connection for other reasons than Wi-Fi access point reboot.
  The issue root cause is not known, but it is highly probable to be related with the usage of Wi-Fi Low Power Mode.
  After the application reboot, the device is always able to re-connect to Wi-Fi network and operate normally.

  **Affected platforms:** nRF7002

  **Workaround:** Disable Wi-Fi Low Power mode for your application by setting :kconfig:option:`CONFIG_NRF_WIFI_LOW_POWER` to ``n`` in the application :file:`prj.conf`.
  
.. rst-class:: v2-5-0

KRKNWK-17925: The nRF Toolbox application for iOS cannot control :ref:`matter_lock_sample` using NUS
  The nRF Toolbox application sends one additional character in all NUS commands, so they are not correctly parsed by the :ref:`matter_lock_sample`.
  The issue was observed only on the nRF Toolbox 5.0.9 version on iOS system. 

  **Workaround:** Use nRF Toolbox for iOS version other than 5.0.9 or any version of nRF Toolbox for Android.

.. rst-class:: v2-5-0

KRKNWK-17914: The links to Kconfig options from :file:`Kconfig.features` do not work in |NCS| documentation 
  The links to all Kconfig options defined in :file:`modules/lib/matter/config/nrfconnect/Kconfig.features` file do not work in documentation. 
```

Author: Updated. Please check.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1

Reviewer 4: It is not valid for 2.5.0 anymore.
```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1
```

Author: Updated for issue KRKNWK-17718

KRKNWK-17718: Failure at TC-DGSW-1.1 Matter Certification test
  The issue happens due to a wrong Software Diagnostic cluster feature map with features enabled by default.

  **Workaround:** Set the default value of the Software Diagnostic cluster feature map to 0 using the ZAP Tool and regenerate files.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Reviewer 4: It is not valid for 2.5.0 anymore.
```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0
```

Author: Updated for issue KRKNWK-17699

KRKNWK-17699: Failure at TC-BINFO-1.1 Matter Certification test for the lock sample
  The issue happens due to a noncompliant Basic Information cluster revision for Matter 1.1.0 in the lock sample.

  **Workaround:** Set the default value of the Basic Information cluster revision to 1 in :file:`samples/matter/lock/src/lock.zap` using the ZAP Tool and regenerate files.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

Reviewer 4: It is not valid for 2.5.0 anymore.
```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
```

Author: Updated for issue KRKNWK-17594

KRKNWK-17594: Application does not always respond when forcing fail-safe cleanup
  This can happen because of the Thread interface being unnecessarily reset and can result in the TC-CNET-4.10 Matter Certification test failing.

  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``3b2d8e1367d9055a78d72365323cfbf60e054975``).

> .. rst-class:: v2-5-0 v2-4-2 v2.4.1 v2-4-0 v2-3-0 v2-2-0

Reviewer 4: It is not valid for 2.5.0 anymore.
```suggestion
.. rst-class:: v2-4-2 v2.4.1 v2-4-0 v2-3-0 v2-2-0
```

Author: Updated for issue KRKNWK-17535

The issues in this section are related to the HomeKit protocol.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

Reviewer 5: Fixed in NCS 2.5.0

Author: Updated for Issue KRKNWK-17343.

The issues in this section are related to the :ref:`serial_lte_modem` application.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0

Reviewer 6: ```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
```
Serial LTE Modem no longer has secure_bootloader overlay.

Author: Updated.

The issues in this section are related to the :ref:`serial_lte_modem` application.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

NCSDK-19278: Modem traces captured through UART are corrupted if RTT logs are simultaneously captured.
  When capturing modem traces through UART with `Cellular Monitor`_ app and simultaneously capturing RTT logs, for example, with J-Link RTT Viewer, the modem trace misses packets, and captured packets might have incorrect information.

  **Affected platforms:** nRF9160, nRF9161

  **Workaround:** If you need to capture modem traces and RTT logs at the same time, add the following change to :file:`nrf9160dk_nrf9160_ns.overlay` or :file:`nrf9161dk_nrf9161_ns.overlay`, depending on the board you are using.
  Otherwise, you can choose not to capture RTT logs.
  Having RTT logs enabled does not cause this issue.

  .. parsed-literal::
     :class: highlight

     &uart1 {
      hw-flow-control;
     };

  This increases the overall power consumption even when Serial LTE Modem is in sleep mode.

> 

Reviewer 6: ```suggestion

.. rst-class:: v2-5-0

NCSDK-19278: UART hardware flow control not working for some scenarios with Thingy:91
  After we run out of UART receive buffers and hardware flow control is enabled, we can sometimes still receive ``UART_RX_STOPPED`` with overflow status and lose data that is sent to Thingy:91.
  This depends on timing and what has been transmitted previously.

  **Affected platforms:** Thingy:91

  **Workaround:** Use the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to ensure that there is adequate buffer space for the worst case scenario.
  Additionally, you may disable harware-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.

```

Author: Updated.

  **Affected platforms:** nRF9160, nRF9161

  **Workaround:** If you need to capture modem traces and RTT logs at the same time, add the following change to :file:`nrf9160dk_nrf9160_ns.overlay` or :file:`nrf9161dk_nrf9161_ns.overlay`, depending on the board you are using.
  Otherwise, you can choose not to capture RTT logs.
  Having RTT logs enabled does not cause this issue.

  .. parsed-literal::
     :class: highlight

     &uart1 {
      hw-flow-control;
     };

  This increases the overall power consumption even when Serial LTE Modem is in sleep mode.

.. rst-class:: v2-5-0

NCSDK-19278: UART hardware flow control is not working for some scenarios with Thingy:91
  After we run out of UART receive buffers and hardware flow control is enabled, we can sometimes still receive ``UART_RX_STOPPED`` with overflow status and lose data that is sent to Thingy:91.
  This depends on timing and what has been transmitted previously.

  **Affected platforms:** Thingy:91

  **Workaround:** Use the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to ensure that there is adequate buffer space for the worst-case scenario.

>   Additionally, you may disable harware-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.

Reviewer 6: ```suggestion
  Additionally, you may disable hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.
```

Reviewer 6: Sorry about this.

  **Workaround:** If you need to capture modem traces and RTT logs at the same time, add the following change to :file:`nrf9160dk_nrf9160_ns.overlay` or :file:`nrf9161dk_nrf9161_ns.overlay`, depending on the board you are using.
  Otherwise, you can choose not to capture RTT logs.
  Having RTT logs enabled does not cause this issue.

  .. parsed-literal::
     :class: highlight

     &uart1 {
      hw-flow-control;
     };

  This increases the overall power consumption even when Serial LTE Modem is in sleep mode.

.. rst-class:: v2-5-0

NCSDK-19278: UART hardware flow control is not working for some scenarios with Thingy:91
  After we run out of UART receive buffers and hardware flow control is enabled, we can sometimes still receive ``UART_RX_STOPPED`` with overflow status and lose data that is sent to Thingy:91.
  This depends on timing and what has been transmitted previously.

  **Affected platforms:** Thingy:91

  **Workaround:** Use the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to ensure that there is adequate buffer space for the worst-case scenario.
  Additionally, you may disable hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.

> 

Author: Updated. Please check.

> 

Author: Updated. Please check.

The issues in this section are related to the :ref:`serial_lte_modem` application.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

> NCSDK-19278: Modem traces captured through UART are corrupted if RTT logs are simultaneously captured.

Reviewer 6: ```suggestion
NCSDK-20457: Modem traces captured through UART are corrupted if RTT logs are simultaneously captured.
```

  When capturing modem traces through UART with `Cellular Monitor`_ app and simultaneously capturing RTT logs, for example, with J-Link RTT Viewer, the modem trace misses packets, and captured packets might have incorrect information.

  **Affected platforms:** nRF9160, nRF9161

  **Workaround:** If you need to capture modem traces and RTT logs at the same time, add the following change to :file:`nrf9160dk_nrf9160_ns.overlay` or :file:`nrf9161dk_nrf9161_ns.overlay`, depending on the board you are using.
  Otherwise, you can choose not to capture RTT logs.
  Having RTT logs enabled does not cause this issue.

  .. parsed-literal::
     :class: highlight

     &uart1 {
      hw-flow-control;
     };

  This increases the overall power consumption even when Serial LTE Modem is in sleep mode.

> .. rst-class:: v2-5-0
> 
> NCSDK-19278: UART hardware flow control is not working for some scenarios with Thingy:91
>   After we run out of UART receive buffers and hardware flow control is enabled, we can sometimes still receive ``UART_RX_STOPPED`` with overflow status and lose data that is sent to Thingy:91.
>   This depends on timing and what has been transmitted previously.
> 
>   **Affected platforms:** Thingy:91
> 
>   **Workaround:** Use the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to ensure that there is adequate buffer space for the worst-case scenario.
>   Additionally, you may disable hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.

Reviewer 6: ```suggestion
.. rst-class:: v2-5-0

NCSDK-19278: SLM attempts to use UART hardware flow control even though Connectivity Bridge does not support it
  With Thingy:91, Connectivity Bridge in nRF52840 terminates the USB traffic and sends the traffic via UART to SLM in nRF9160. Connectivity Bridge does not enable UART hardware flow control and since both ends need to enable it, SLM should not enable it either.
  Without hardware flow control, the buffer sizes must be set according to worst-case scenario. In this case, SLM uses the default buffer size of 3 x 256 bytes and will drop incoming UART traffic after the buffers are full.

  **Affected platforms:** Thingy:91

  **Workaround:** Set the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to 2048 to ensure that there is adequate buffer space to receive traffic from the Connectivity Bridge and disable the hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.
  If even larger buffer spaces are required, the :kconfig:option:`CONFIG_BRIDGE_BUF_SIZE` Kconfig option for Connectivity Bridge and the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option for SLM, should be set accordingly.
```

.. note::
    nRF Desktop is also affected by the Bluetooth LE issue :ref:`NCSDK-19865 <ncsdk_19865>`.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0

NCSDK-8304: HID configurator issues for peripherals connected over Bluetooth® LE to Linux host
  Using :ref:`nrf_desktop_config_channel_script` for peripherals connected to host directly over Bluetooth LE might result in receiving improper HID feature report ID.
  In such case, the device will provide HID input reports, but it cannot be configured with the HID configurator.

  **Workaround:** Use BlueZ in version 5.56 or higher.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0

Author: Updated.

NCSDK-22953: A HID output report interrupts scanning in the :ref:`nrf_desktop_ble_scan` even if there are no peripherals connected
  Connecting a HID dongle over USB to a HID host might lead to submitting a HID output report and stopping Bluetooth LE scanning even if there are no peripherals connected through the dongle.
  In this case, scanning stop delays establishing a connection with peripherals.

  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``b4860f7c9475ff8e61a4c3b907968987bb6311bd``).

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0

Author: Updated

.. note::
    nRF Desktop is also affected by the Bluetooth LE issue :ref:`NCSDK-19865 <ncsdk_19865>`.

> .. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0

Reviewer 7: I think we can add `wontfix` tag here too.

Author: Updated. Please check.

Reviewer 7: I think you accidentally added it to the `NCSDK-19865: GATT Robust Caching issues for bonded peers` instead of adding it here. Please fix :)

Reviewer 7: Fixed.

The issues in this section are related to the :ref:`nrf53_audio_app` application.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Reviewer 8: I think the issue numbers should be in rising order? (going from highest to lowest OCT number)

Reviewer 8: Hmm, for Audio the known issues are ordered by rising OCT ticket number in e.g. the NCS 2.4.0 release tag. It makes it easier for us to know which known issue has been added last.
It is not critical, but it would be great if we could keep this structure.

Author: Updated. Please check.

The issues in this section are related to the :ref:`nrf53_audio_app` application.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Reviewer 8: New known issues for nRF5340 Audio:

- - -
Affected versions: NCS 2.5.0
OCT-2766: BIS headset might get into an infinite loop

If a BIS headset is paused, and the BIS broadcaster is reset, then if the headset is unpaused, it will end up in an infinite loop.
- - -
Affected versions: NCS 2.5.0
OCT-2765: BIS headset stack overflow if gateway is periodically reset

A BIS headset might get stack overflow if the BIS source is reset several times with short intervals.

Author: Updated. Please check. Also updated the affected versions, wherever applicable.

Reviewer 8: I cannot see that OCT-2767 is added.
I also modified the text a little for these new issues at the end of Friday. It would be great to use this.

> 

Author: Updated. Please check.

  **Workaround:** Use an alternative ACL connection interval.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

OCX-153: Cannot create BIG sync after terminate pending BIG sync
  If a pending BIG sync is canceled by sending the LE Broadcast Isochronous Group Sync Established command, it is impossible for the host to create a new BIG sync afterwards.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

OCX-155: Larger timestamps than intended

>    The timestamps/Service Data Unit references (SDU refs), may occasionally be larger than intended and then duplicated in the next interval.

Author: Updated. Please check.

The following known issues apply to the LE Audio subsystem (NET core controller) for nRF5340 used in the nRF5340 Audio application.

> .. rst-class:: v2-5-0

Reviewer 8: New issue for Controller subsystem for nRF5340 Audio:

- - -
Affected versions: NCS 2.5.0
OCT-2761: Controller asserts on CIS gateway if two paused headsets reset

If a CIS headset pair is paused, then reset simultaneously a few times while remaining paused, the controller on gateway might get unresponsive.

Author: Updated. Please check.

Reviewer 8: I wrote the wrong ticket number here. It should be OCX-223. Apologize

Author: Updated. Please check.

  **Workaround:** Use different :kconfig:option:`CONFIG_BT_DEVICE_NAME` for BIS and CIS.

.. rst-class:: v2-5-0

OCT-2712: Gateway cannot properly handle a connected headset if it continues to advertise
  The gateway will try to connect to a headset that is already connected.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-5-0

OCT-2713: Gateway can only read one PAC record from each headset
  Some headsets might have several PAC records containing different supported configurations.
  Then the most optimal configuration might not be used, or the gateway might not be able to configure a headset since it does not know about all supported configurations.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-5-0

OCT-2715: There can be a state mismatch between streaming and media control in certain scenarios
  For example, if a stream is in the playing state but media control is in the pause state, it might not be possible to play or pause.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

> OCT-2551: GPIO pin forwarding to the network core is not done for the **RGB2** LED pins

Reviewer 8: @divipillai Last thing: OCT-2551 should be over OCT-2558

  **Workaround:** Enable the :kconfig:option:`CONFIG_BT_SETTINGS` Kconfig option in the project configuration file (:file:`prj_minimal.conf`).

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Author: Updated. Please check.

Bluetooth samples
=================

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Author: Updated

  **Workaround:** Enable the legacy LLCP mechanism (:kconfig:option:`CONFIG_BT_LL_SW_LLCP_LEGACY`).

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0

Author: Updated

  The TX power for a connection cannot be less than -20 dBm.
  Controller still output -20 dBm if the setting is -40 dBm.

> .. rst-class:: v2-5-0

Reviewer 8: We discovered last minute that OCX-188 has re-occurred, so we need to add v2-5-0 to the rst-class on this one.
We have also created a new OCX ticket (OCX-227), since OCX-188 was fixes, but then it broke again.
Is it possible to write it like this?:
OCX-184/OCX-227: If 0 dBm TX power is selected, the FEM/PA TX/RX pins do not toggle correctly

Author: I will add them.

Author: Updated. Please check.

Author: Updated. Please check.

  **Workaround:** Enable the :kconfig:option:`CONFIG_BT_SETTINGS` Kconfig option in the project configuration file (:file:`prj_minimal.conf`).

.. rst-class:: v2-3-0 v2-2-0

NCSDK-18112: :ref:`bluetooth_central_dfu_smp` sample cannot do discovery on the :ref:`smp_svr_sample`
  The :ref:`bluetooth_central_dfu_smp` sample cannot perform the GATT discovery on a DK with the :ref:`smp_svr_sample`.

  **Workaround:** Enable the legacy LLCP mechanism (:kconfig:option:`CONFIG_BT_LL_SW_LLCP_LEGACY`).

.. rst-class:: wontfix v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0

> NCSDK-19942: HID samples do not work with Android 13
>   The :ref:`peripheral_hids_keyboard` and :ref:`peripheral_hids_mouse` samples enter a connection-disconnection loop when you try to connect to them from a device that is running Android 13.
> 

Reviewer 9: Can we generalize this known issue and indicate that this issue affects all NCS releases as this is an Android bug?

```suggestion
NCSDK-19942: HID samples do not work with Android 13
  All Bluetooth samples and applications that are set up for the HIDS use case and have the Bluetooth Privacy feature enabled (:kconfig:option:`CONFIG_BT_Privacy`) enter a connection-disconnection loop when you try to connect to them from a device that is running Android 13.
  
See the list of affected samples and applications:

* :ref:`peripheral_hids_keyboard`
* :ref:`peripheral_hids_mouse`
* :ref:`fast_pair_input_device`
* Fast Pair configurations of the :ref:`nrf_desktop` application
```

FYI: @grochu, @KAGA164, @alstrzebonski, @MarekPieta

Reviewer 10: It is ok to me to have one common entry in Known issue about Android 13

Reviewer 9: @grochu, @KAGA164, are you sure that the HIDS keyboard sample does not work with Android 13? The keyboard sample does enable the CONFIG_BT_PRIVACY option and should work correctly

Reviewer 10: You are right. It should be only peripheral hids mouse sample

  **Workaround:** Enable the :kconfig:option:`CONFIG_BT_SETTINGS` Kconfig option in the project configuration file (:file:`prj_minimal.conf`).

.. rst-class:: v2-3-0 v2-2-0

NCSDK-18112: :ref:`bluetooth_central_dfu_smp` sample cannot do discovery on the :ref:`smp_svr_sample`
  The :ref:`bluetooth_central_dfu_smp` sample cannot perform the GATT discovery on a DK with the :ref:`smp_svr_sample`.

  **Workaround:** Enable the legacy LLCP mechanism (:kconfig:option:`CONFIG_BT_LL_SW_LLCP_LEGACY`).

.. rst-class:: wontfix v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0

NCSDK-19942: HID samples do not work with Android 13

>   Bluetooth samples and applications that are set up for the HIDS use case and have the Bluetooth Privacy feature enabled (:kconfig:option:`CONFIG_BT_Privacy`) enter a connection-disconnection loop when you try to connect to them from a device that is running Android 13.

Reviewer 9: The behaviour depends on the smartphone manufacturer:
- Samsung Galaxy models (connection-disconnection loop)
- Google Pixel models (disconnection after a short period)

```suggestion
  Bluetooth samples and applications that are set up for the HIDS use case and have the Bluetooth Privacy feature enabled (:kconfig:option:`CONFIG_BT_Privacy`) disconnect after a short period or enter a connection-disconnection loop when you try to connect to them from a device that is running Android 13.
```

  See the following list of affected samples and applications:

  * :ref:`peripheral_hids_mouse`
  * :ref:`fast_pair_input_device`
  * Fast Pair configurations of the :ref:`nrf_desktop` application

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

Reviewer 9: Please align the list of releases as the newly added Fast Pair sample is present in NCS from v2.0.0

```suggestion
.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Author: Updated for issue NCSDK-18518

.. rst-class:: v2-3-0

NCSDK-20967: The :ref:`nrf_rpc_entropy_nrf53` sample does not work on the network core
  The network core will not work due a hard fault.

  **Affected platforms:** nRF5340

Other samples
=============

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0

> NCSDK-20046: IPC service sample does not work with ``nrf5340dk_nrf5340_cpuapp``

Reviewer 11: was fixed for 2.4.0

Cellular samples
================

.. rst-class:: v2-5-0

> IRIS-7398: MCUBoot secondary partition in external flash fails for the nRF9161 DK

Reviewer 12: ```suggestion
IRIS-7398: The :ref:`nrf_cloud_multi_service` sample does not support using the MCUBoot secondary partition in external flash fails on the nRF9161 DK.
```

Author: Updated. Please check. We don't use a period at the end for the Known Issue headings.

Reviewer 12: Looks good -- Using `and it fails on the nRF9161 DK`  instead of `on the nrf9161 DK` is potentially unclear, since it implies both statements are separately true, as opposed to the latter being a qualifier for the former, but it'll probably be alright

Bluetooth libraries and services
================================

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Reviewer 7: Fixed in NCS 2.5

Author: Updated

  The assertion failure replicates in the :ref:`fast_pair_input_device` sample.

  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``a8b668e82837295962348e9e681125c2ae11bb4e``).

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

NCSDK-23682: The Fast Pair Seeker might be unable to bond again after losing the bonding information multiple times
  If the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option is disabled on the Fast Pair Provider side, the Fast Pair Seeker that uses the RPA address to connect with the Provider might be unable to bond again after losing the bonding information multiple times.
  The issue is because of `Zephyr issue #64042 <https://github.com/zephyrproject-rtos/zephyr/issues/64042>`_.

  **Workaround:** Keep the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option enabled.

Other libraries
===============

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-1 v1-0-0 v0-4-0

Reviewer 10: Fixed in this release

Author: Updated.

The issues in this section are related to :ref:`Bootloaders <app_bootloaders>`.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-0 v2-4-1

Author: Updated.

NFC
===

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0

> NCSDK-22799: Assert when requesting clock from the NFC interrupt context.

Reviewer 3: ```suggestion
NCSDK-22799: Assert when requesting clock from the NFC interrupt context
```

The issues in this section are related to :ref:`nrfxlib:crypto`.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0

Author: Updated.

NFC
===

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0

Reviewer 10: Fixed in v2.5

Author: Updated.

> 

Reviewer 13: Here the below known issues can be added:

.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
DRGN-19460: The continuous extended scanner cannot receive the AUX_ADV_IND packet
  This can happen if the time between the ADV_EXT_IND and AUX_ADV_IND was more than 840 μs.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-19580: The stack would dereference a NULL pointer
  This can happen if a resolvable :c:enum:`own_address_type` was used in the HCI Le Extended Create Connection V2 command while the resolving list was empty.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-19623: The HCI Reset would not reset the channel map
  The HCI Reset command would not clear the channel map set by the host using the HCI Le Set Host Channel Classification command.
 
.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-18411: The Peer_Address_Type parameter in the LE Connection Complete event was incorrectly set to 2 or 3 
  This can happen when the connection was established to a device whose address was resolved. The least significant bit of the Peer_Address_Type parameter was set correctly.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-20118: The stack would assert when creating advertisers
  This can happen if trying to set up more advertisers than there are available advertising sets.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-20085: The stack would assert when enabling an advertising set
  Enabling an extended advertising set would assert in cases where a host-provided address was not needed and no address had been set up for the advertising set.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-20578: The stack would assert when receiving a non-compliant LL_PHY_RSP
  The controller acting as a central would assert when receiving a non-compliant LL_PHY_RSP from a peer device.

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

> .. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

Reviewer 13: .. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0

Author: Updated.

DRGN-18568: Using :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option lowers the value of radio output power
  The actual value is lower than the default one in case the :kconfig:option:`CONFIG_BT_CTLR_TX_PWR_ANTENNA` or :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig options are used together with the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

Reviewer 13: Since the issue has been already fixed in NCS 2.4 actually, change to the below
 
rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0

DRGN-19460: The continuous extended scanner cannot receive the ``AUX_ADV_IND`` packet
  This can happen if the time between the ``ADV_EXT_IND`` and ``AUX_ADV_IND`` is more than 840 μs.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0

DRGN-19580: The stack would dereference a ``NULL`` pointer
  This can happen if a resolvable :c:enum:`own_address_type` is used in the HCI Le Extended Create Connection V2 command while the resolving list is empty.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0

DRGN-19623: The HCI Reset would not reset the channel map
  The HCI Reset command would not clear the channel map set by the host using the HCI Le Set Host Channel Classification command.

> .. rst-class:: v2-4-2 v2-4-1 v2-4-0

Reviewer 13: Sorry, here should also cover v2-3-0

Author: Updated for Issue DRGN-18411.

NCSDK-22818: TF-M does not build with :kconfig:option:`CONFIG_TFM_PROFILE_TYPE_MINIMAL` disabled and :kconfig:option:`CONFIG_TFM_LOG_LEVEL_SILENCE` enabled
  This fails with the error: message "undefined reference to 'end'" caused by printf being included in the build.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0

Reviewer 14: double-checked. This feedback looks correct to me.

I proposed removing NCSDK-22629 from affected versions.
Joakim has found that NCSDK-22629 was the wrong jira issue and has a
PR for fixing this.

My proposal is valid because the NCS text says that nrfjprog
10.22.x is affected, but 10.23.x is the latest version and I
assume that is what is installed and supported.

Georgios proposes adding it to the affected version but I don't think
that it is correct.

     nrfjprog -f nrf91 --debugreset

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

Reviewer 14: This feedback was wrong. I looked at the wrong text.

I was not accidentally correct either.

This change should be reverted.

We are affected by NCSDK-18321.

  * Increase the TF-M partition :kconfig:option:`CONFIG_PM_PARTITION_SIZE_TFM`.
  * Use Zephyr SDK version 0.14.1.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

Reviewer 14: ```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

NCS 2.5.0 uses an unaffected version of the "Zephyr SDK".

Author: Updated

Reviewer 14: The reasoning in this feedback is wrong. I looked at the wrong text.

But I was accidentally correct.

So we should keep this proposed change.

We are not affected by NCSDK-16916.

  **Workaround:** Add the text ``zephyr_link_libraries(-Wl,--undefined=my_missing_debug_symbol)`` in the application's :file:`CMakeLists.txt` file.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

NCSDK-12342: Potential SecureFault exception while accessing protected storage
  When accessing protected storage, a SecureFault exception is sometimes triggered and execution halts.

> .. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0

Reviewer 14: ```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
```

Author: updated

Reviewer 14: This this feedback is wrong. I looked at the wrong text.

And I was not accidentally correct either.

So we should revert this change.

We are affected by NCSDK-11195.
